Tests/ExporterTestsV4.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Xunit;
```

Comment:
Git commit logs first line is 'subject', then a newline and then the body. I think you should put the 'Had to change...' part in the commit log body.